doc/source/conf.py

```diff
@@ -48,17 +48,17 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'psqlgraph'
-copyright = u'2014, The University of Chicago'
+project = 'psqlgraph'
+copyright = '2019, The University of Chicago'
```

Comment:
doh.  sam-i-am crossed the year boundary.  we'll need to update this to 2020 in thidwick.